### ISSUE TITLE
Feature Request: Save buffer to file

### ISSUE THREAD

**Issue by aoyama-val** [noborus/ov#418]
I am using ov as the pager of psql, and sometimes I would like to save the result of the query to a file.
Is it possible to add "Save to file" like less's `-O` and `s`?

**Comment by noborus** [comment]
Thank you for the issue.
I'm thinking of implementing a save function.

**Comment by noborus** [comment]
Even in the current version you can also save using a redirect.

```psql
\setenv PSQL_PAGER 'ov -F -C -d "|" -H1 --column-rainbow --column-mode >address.txt'
SELECT * FROM address;
```

**Comment by aoyama-val** [comment]
It's not that I want to save every result. Usually, after seeing the result, I decide if I should save it.
I believe this is a common workflow when using the pager for output of a command.
For example, suppose you want to use ov for git and you set GIT_PAGER as you described. Then you would get the output file every time you invoke git. This would be so annoying.

**Comment by noborus** [comment]
yes. I'm trying to implement another.
However, psql's `\setenv` can be used to temporarily set things.

**Comment by noborus** [comment]
Implemented in v0.32.0. thank you.
